pkg/geo/wkt/lex.go

```diff
@@ -36,21 +36,28 @@ type ParseError struct {
 	problem string
 	pos     int
 	str     string
+	hint    string
 }
 
 func (e *ParseError) Error() string {
-	return fmt.Sprintf("%s at pos %d\n%s\n%s^", e.problem, e.pos, e.str, strings.Repeat(" ", e.pos))
+	// TODO(ayang) rethink position marker since input can be multiple lines, maybe use a "<-"?
+	err := fmt.Sprintf("%s at pos %d\n%s\n%s^", e.problem, e.pos, e.str, strings.Repeat(" ", e.pos))
+	if e.hint != "" {
+		err += fmt.Sprintf("\nhint: %s", e.hint)
```

Comment:
nit: use HINT:

we usually use `errors.WithHintf`, which properly sets this on postgres' error protocol, but since this is meant to generic i am ok ignoring this for now!